### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by TenkyuChimata**
Add Chinese translation

### CODE REVIEW

File: ZH.txt [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+距离最近的地震优先
+
+# Largest earthquake first
+震级最大的地震优先
+
+# Strongest close-by earthquake first, relative to device's location
+震级大且距离最近的地震优先
+
+# #Earthquake recorded on the #RaspberryShake #CitizenScience seismic network. See what's shaking near you with the @raspishake #ShakeNet mobile app
+#Earthquake recorded on the #RaspberryShake #CitizenScience seismic network. See what's shaking near you with the @raspishake #ShakeNet mobile app
```
Comment (trinhtuanvu) [inline ZH.txt]: Missing translation here.
```suggestion
#RaspberryShake #CitizenScience地震网络上记录的＃地震。 使用@raspishake #ShakeNet移动应用程序查看附近正在发生什么
```

File: ZH.txt [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+
+# #Earthquake event recorded on the #RaspberryShake #CitizenScience seismic network. See what's shaking near you with the @raspishake #ShakeNet mobile app
+#Earthquake event recorded on the #RaspberryShake #CitizenScience seismic network. See what's shaking near you with the @raspishake #ShakeNet mobile app
```
Comment (trinhtuanvu) [inline ZH.txt]: Missing translation here.
```suggestion
#RaspberryShake #CitizenScience地震网络上记录的#Earthquake事件。 使用@raspishake #ShakeNet移动应用程序查看附近正在发生什么
```

File: ZH.txt [hunk 3]
```diff
@@ -0,0 +1,468 @@
...
+
+# 24-hr Plot data recorded on the #RaspberryShake #CitizenScience seismic network. See what's shaking near you with the @raspishake #ShakeNet mobile app
+#RaspberryShake #CitizenScience seismic network. See what's shaking near you with the @raspishake #ShakeNet mobile app
```
Comment (trinhtuanvu) [inline ZH.txt]: Missing translation here.
```suggestion
＃24小时制图数据记录在#RaspberryShake #CitizenScience地震网络上。 使用@raspishake #ShakeNet移动应用程序查看附近正在发生什么
```

**Comment by TenkyuChimata** [comment]
Ok, done.

**Pull request merged**